packages/labs/src/components/popover/arrow.tsx

```diff
@@ -0,0 +1,25 @@
+/*
+ * Copyright 2017 Palantir Technologies, Inc. All rights reserved.
+ * Licensed under the BSD-3 License as modified (the “License”); you may obtain a copy
+ * of the license at https://github.com/palantir/blueprint/blob/master/LICENSE
+ * and https://github.com/palantir/blueprint/blob/master/PATENTS
+ */
+
+import * as React from "react";
+import { Arrow } from "react-popper";
+
+import { Classes } from "@blueprintjs/core";
+
+const SVG_SHADOW_PATH = "M8.11 6.302c1.015-.936 1.887-2.922 1.887-4.297v26c0-1.378" +
+    "-.868-3.357-1.888-4.297L.925 17.09c-1.237-1.14-1.233-3.034 0-4.17L8.11 6.302z";
+const SVG_ARROW_PATH = "M8.787 7.036c1.22-1.125 2.21-3.376 2.21-5.03V0v30-2.005" +
+    "c0-1.654-.983-3.9-2.21-5.03l-7.183-6.616c-.81-.746-.802-1.96 0-2.7l7.183-6.614z";
+
+export const PopoverArrow: React.SFC<{ angle: number }> = ({ angle }) => (
```

Comment:
yo i have no idea.

_update_: turns out it's `0`. great. i moved the function that computes angle into this file, cuz it makes sense for those two to live together